test/ffprobe_test.go

```diff
@@ -0,0 +1,72 @@
+package ffprobe_test
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	ffprobe "github.com/vansante/go-ffprobe"
+)
+
+func Test_ffprobe(t *testing.T) {
+	// test GetProbeData
+	path := "test.mp4"
+	data, err := ffprobe.GetProbeData(path, 500*time.Millisecond)
+	if err != nil {
+		t.Errorf("Error getting data: %v", err)
+	}
+
+	// test data
+	_, err = json.MarshalIndent(data, "", "  ")
```

Comment:
Im not sure you are why testing the marshalling of the data struct, marshalling is not covered by this package right?